tasks/e2e.sh

```diff
@@ -178,5 +182,10 @@ npm test -- --watch=no
 # Test the server
 npm start -- --smoke-test
 
+# Test special scripts versions
+create_react_app --scripts-version=0.4.0 test-app-version-number
+create_react_app --scripts-version=https://registry.npmjs.org/react-scripts/-/react-scripts-0.4.0.tgz test-app-tarball-url
+create_react_app --scripts-version=react-scripts-fork test-app-fork
```

Comment:
We should at least test *something*. Perhaps that the created project contains the corresponding fork in `node_modules`.